gradle.properties

```diff
@@ -25,6 +25,7 @@ stdlibSqlVersion=1.7.0
 
 # Level 09
 stdlibPersistVersion=0.1.0
+stdlibPersistSnaptshotVersion=0.2.0-SNAPSHOT
```

Comment:
```suggestion

# Persist native jar maven dependency
persistNativeVersion=0.2.0-SNAPSHOT
```

Do nout use stdlib prefix for this. This will then be picked up by dependendcy workflows.